src/app/settings/organization-plans.component.html

```diff
@@ -140,7 +140,7 @@ <h2 class="spaced-header">{{'summary' | i18n}}</h2>
                 <ng-container *ngIf="selectablePlan.isAnnual">
                     {{'annually' | i18n}}
                     <small *ngIf="selectablePlan.basePrice">
-                        {{'basePrice' | i18n}}: {{selectablePlan.basePrice / 12 | currency:'$'}} &times;12
+                        {{'basePrice' | i18n}}: {{selectedPlanInterval | currency:'$'}} &times;12
```

Comment:
If this is `"Annual"` or `"Monthly"`, I'm not seeing how that translates into a base price or a currency.

Reply:
Yeah, I had gotten a little carried away with that property refactor. Already fixed in latest.